### ISSUE TITLE
Rewards extension background pages should not be running when Rewards is disabled

### ISSUE THREAD

**Issue by tmancey** [brave/brave-browser#5429]
Rewards extension background pages should not be running when Rewards is disabled. The extension itself is loaded to display the browser action, but the background page doesn't need to be running as this will have the benefit of less memory usage.

Another way to address this would be to have a different extension for the background page The background page can be modified to be compatible with non-persistent background pages, but a second extension might be easier.

Basically a "dummy" extension just to display the rewards icon when rewards isn't enabled. Or keep the existing extension as-is, but move the background page and update the background page to accept requests from that extension id which is easier.

**Comment by shark0der** [comment]
Came here looking for this specific issue. I like Brave's idea to make a privacy-oriented browser but shoving in rewards and other trash does not inspire this.

You can see that there are 3 processes that are running even though I've disabled Brave Rewards:
- Brave rewards
- Bat Ledger Service
- Bat Ads Service

![Brave task manager](https://i.imgur.com/PgXx9Zh.png)

Would love to see this gone. Thank you.

**Comment by NejcZdovc** [comment]
+1 https://github.com/brave/brave-browser/issues/6972

**Comment by bughit** [comment]
@tmancey since this issue is focused on the rewards extension should it not be closed? The rewards process is gone.

**Comment by shark0der** [comment]
@bughit the "Extension: Brave Rewards" process is indeed gone. However, I can still see the "Bat Ledger Service" and "Bat Ads Service".

**Comment by bughit** [comment]
@shark0der 

This issue (it's title and description) is narrowly focused on the rewards extension. Unless the title and the description are updated to explicitly widen the scope, it would have to be considered fixed and there's a good chance someone will eventually close it, exactly as happened with #3436 which was also intended to have wider scope (#4612 was duped to it, but never addressed, and this pattern is repeating with this one).

@tmancey, please either widen the scope (per #6972) or close this.

**Comment by eljuno** [comment]
+1 from Community https://community.brave.com/t/how-to-stop-bat-ads-service/116242/4?u=eljuno

**Comment by AshPee** [comment]
I am new here so please be gentle.
I can't tell after reading all of the above just where exactly Brave is timeline wise on this/my issue. 
Can someone please tell me in plain-speak when the following will be fixed: 

_Can't stop Bat Ads Service and Bat Ledger Service from running
when I use Brave... I realized today I can use good old Process Explorer from Sysinternals to suspend these Brave services each time I start Brave. But that is a big pain and I think not really good enoug_h.

**Comment by NejcZdovc** [comment]
closing as this will be needed in https://github.com/brave/brave-browser/issues/9130

**Comment by bughit** [comment]
@NejcZdovc Please clarify, if one is not using rewards why is it needed?

**Comment by NejcZdovc** [comment]
@bughit we will add more context in the issue and blog post
